apps/frontend/src/components/Preview/PreviewDroppable/PreviewDroppable.tsx

```diff
@@ -7,7 +7,15 @@ import PreviewDroppableStyle from './PreviewDroppable.module.css';
 const PreviewDroppable: React.FunctionComponent = () => {
     const dispatch = useDispatch();
     const [isHovered, setIsHovered] = useState<boolean>(false);
-
+    const [isDraggabled, setIsDraggisDraggabled] = useState<boolean>(false);
```

Comment:
Check your variable names before PRs 😂 
Should be replaced by `isDraggedOver` and `setIsDraggedOver`.